src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -307,7 +307,10 @@ private void ReAddReorganisedTransactions(Block? previousBlock)
                         continue;
                     }
                     _hashCache.Delete(tx.Hash!);
-                    SubmitTx(tx, isEip155Enabled ? TxHandlingOptions.None : TxHandlingOptions.PreEip155Signing);
+                    // For reorg we will submit with PersistentBroadcast
+                    SubmitTx(tx, isEip155Enabled ?
+                        TxHandlingOptions.PersistentBroadcast :
+                        TxHandlingOptions.PersistentBroadcast | TxHandlingOptions.PreEip155Signing);
```

Comment:
why persistent broadcast?

Reply:
It doesn't really give boost for inclusion on this node, just sends to other nodes more aggresively

Reply:
I'm not a fan of the idea of adding reorged transactions to the persistent broadcast. We had this feature a few years ago but removed it as an optimization.
Persistent broadcast is a strong booster for local transactions, and I don't think that we should care about reorged transactions that much.
That said, I'm not opposed to implementing it if you have a strong preference - I just personally prefer not to.

Reply:
Removed the PersistentBroadcast